ClimsoftVer4/ClimsoftVer4/formMetadata.vb

```diff
@@ -1751,4 +1759,35 @@ Err:
         End If
     End Sub
 
+    Private Sub CombEW_SelectedIndexChanged(sender As Object, e As EventArgs) Handles CombEW.SelectedIndexChanged
+        Dim myLon As Double
+        If CombEW.SelectedItem = "W" Then
+
+            myLon = (-1) * (txtDegreesLon.Text + txtMinutesLon.Text / 60 + txtSecondsLon.Text / 3600)
```

Comment:
@isedwards - Is there any wise way of leaving the text box blank instead of default these to 0 ? I would wish to default the Combo boxes alone.